tests/core/test_config.py

```diff
@@ -36,57 +35,9 @@
 )
 def test_check_dict_leaves(d_a: dict, d_b: dict, overlap: list) -> None:
     """Checks overlapping dictionary search function."""
-    assert overlap == config.check_dict_leaves(d_a, d_b, [])
+    from virtual_rainforest.core.config import check_dict_leaves
 
-
-@pytest.mark.parametrize(
-    "out_path,expected_log_entries",
-    [
-        (
-            "./complete_config.toml",
-            (
-                (
-                    CRITICAL,
-                    "A config file in the user specified output folder (.) already "
-                    "makes use of the specified output file name (complete_config.toml)"
-                    ", this file should either be renamed or deleted!",
-                ),
-            ),
-        ),
-        (
-            "bad_folder/complete_config.toml",
-            (
-                (
-                    CRITICAL,
-                    "The user specified output directory (bad_folder) doesn't exist!",
-                ),
-            ),
-        ),
-        (
-            "pyproject.toml/complete_config.toml",
-            (
-                (
-                    CRITICAL,
-                    "The user specified output folder (pyproject.toml) isn't a "
-                    "directory!",
-                ),
-            ),
-        ),
-    ],
-)
-def test_check_outfile(caplog, mocker, out_path, expected_log_entries):
```

Comment:
just double-checking, did you replace `check_outfile` with the `save_to_netcdf` functionality?

Reply:
I moved `check_outfile` to `core.utils` (from `core.config`). This is basically because I started using the function in `core.data` as well so I thought it made sense to move it to the utility function module as it does something generally useful rather than something only useful to the `core.config` module

Reply:
This specific test hasn't been deleted it has just been moved to `tests/core/test_utils.py`